frontend/src/components/regions/CardDetails/Presentation.js

```diff
@@ -17,23 +33,47 @@ import ProjectDetails from "./ProjectDetails";
 import UsersTable from "./UsersTable";
 import Reviews from "./Reviews";
 
-const useStyles = makeStyles((theme) => ({
-  paper: {
-    padding: theme.spacing(1, 2, 1),
-  },
+const useStyles = makeStyles((theme) => {
+  const card = {
+    borderWidth: 3,
+    borderRadius: theme.spacing(2),
+    margin: theme.spacing(1),
+    textAlign: "center",
+  };
+  return {
+    paper: {
+      padding: theme.spacing(1, 2, 1),
+    },
+
+    commentColumn: {
+      minWidth: 300,
+    },
 
-  commentColumn: {
-    minWidth: 300,
-  },
+    tableContainer: {
+      maxHeight: 200,
+    },
+    sectionPaper: {
+      padding: theme.spacing(1),
+      marginBottom: theme.spacing(1),
+    },
+    root: {
+      margin: theme.spacing(1),
+      width: theme.spacing(40),
+      height: theme.spacing(16),
+    },
 
-  tableContainer: {
-    maxHeight: 200,
-  },
-  sectionPaper: {
-    padding: theme.spacing(1),
-    marginBottom: theme.spacing(1),
-  },
-}));
+    [BLOCKED]: { ...card, backgroundColor: grey[400] },
+    [READY]: { ...card, backgroundColor: blue[400] },
+    [IN_PROGRESS]: { ...card, backgroundColor: green[400] },
```

Comment:
Are you sure this is DRY? It looks very familiar. 

If a piece of code needs to be reused then copy-paste usually isn't the best bet.  Rather make a little widget we can use  in multiple places